sdk/spring-sdk/src/it/java/com/example/wiring/SpringSdkWiringIntegrationTest.java

```diff
@@ -314,4 +398,16 @@ public void verifyFindUsersByNameStreaming() {
                     .size(),
             new IsEqual(2));
   }
+
+  //TODO add test that
+  // GIVEN view with subscription on type level marked with ignore
+  // to event having two different subtypes,
+  // the view handling only one of them
+  // WHEN another event the can handle comes along
+  // THEN the router delivers the event
+  // WHEN the event not handled comes along
+  // THEN the router ignores the event
+  // WHEN another event the can handle comes along
+  // THEN the router delivers the event
+  // THEN the view has the final state
```

Comment:
missing tests?

Reply:
Nope, this was the placeholder for verifyViewIsNotSubscribedToMultiplyAndRouterIgnores. Deleted the comments